### PR TITLE
Add datetime in conferences cards

### PR DESCRIPTION
**Pull request by ElviaBth**
🎩 What? Why?
Add datetime in conferences cards.

📌 Related Issues
Related to #23 

📷 Screenshots
image
image

♥️ Thank you!

### CODE REVIEW

File: app/cells/decidim/conferences/conference_m/tags.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="card__icondata">
+        <ul class="card-data">
+            <li class="card-data__item">
+                <%= icon "datetime", class: "icon--big", role: "img", aria_label: t("decidim.meetings.meetings.show.date") %>
+            </li>
+            <li class="card-data__item">
+                <div class="card-data__item—multiple">
+                    <div>
+                        <strong>
+                        <%= model.start_time %>
```
Comment (microstudi) [inline app/cells/decidim/conferences/conference_m/tags.erb]:
```suggestion
                        <%= l model.start_date format: :decidim_with_month_name %>
```

Comment (microstudi) [reply app/cells/decidim/conferences/conference_m/tags.erb]: l is a shortcut to format dates, in decidim there's some standard formats (like the one suggested)

File: app/cells/decidim/conferences/conference_m/tags.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                        </strong>
+                    </div>
+                    <svg aria-label="arrow-thin-right" role="img" aria-hidden="true" class="icon--arrow-thin-right icon icon--big muted"><title>arrow-thin-right</title><use href="/decidim-packs/media/images/icons-8973a21949776dfa1985.svg#icon-arrow-thin-right"></use></svg>
+                    <div>
+                        <strong>
+                            <%= model.end_time %>
```
Comment (microstudi) [inline app/cells/decidim/conferences/conference_m/tags.erb]: here the same

File: app/models/concerns/conference_override.rb [hunk 3]
```diff
@@ -10,5 +10,13 @@ module ConferenceOverride
...
       where("(CONCAT(' ', hashtag, ' ') ILIKE ?) OR (CONCAT(' ', hashtag, ' ') ILIKE ?)", "% #{hashtag} %", "% ##{hashtag} %")
     end
+
+    def start_time
+      start_date
+    end
+
+    def end_time
+      end_date
+    end
```
Comment (microstudi) [inline app/models/concerns/conference_override.rb]: no need for this anymore

File: app/cells/decidim/conferences/conference_m/tags.erb (hunk 1 above)
Comment (ElviaBth) [reply app/cells/decidim/conferences/conference_m/tags.erb]: Thanks. Done!

**Review by microstudi — CHANGES_REQUESTED** [review]
Nice job @ElviaBth 
just remove the added methods on the conference override and will be good to go!

File: app/models/concerns/conference_override.rb (hunk 3 above)
Comment (ElviaBth) [reply app/models/concerns/conference_override.rb]: Thanks. Done!

File: app/cells/decidim/conferences/conference_m/tags.erb (hunk 2 above)
Comment (ElviaBth) [reply app/cells/decidim/conferences/conference_m/tags.erb]: Done!

**Review by microstudi — APPROVED** [review]
Great job!

**Pull request merged**